relay/relay-network.md

```diff
@@ -0,0 +1,32 @@
+# Relay Network
+
+Relay network is defined by the federation of relay servers exchanging messages to route from a publishing client to a subscribing client on the same topic.
+
+## Goals
+
+Relay servers will route messages through other servers in the network that also have matching subscribers on a topic whilst maintaing the same expected behavior described by a publisher when requested to its own server
+
+## Iridium Message
+
+An Iridium message is defined as a serialized message of a successfully cached message from a publishing request from a connected client with one of the servers.
+
+- **prefix** - a string prefix ("irn") for exchanged iridium messages.
+- **version** - an integer number of the serialized schema for messages.
+- **tag** - an integer label that identifies a message without knowing its contents.
```

Comment:
Change from api to tag should be applied on the client side as well in the Policy object